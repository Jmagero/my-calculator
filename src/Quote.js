/* eslint-disable */
import React from 'react';
const Quote = () => {
    return(
        <React.Fragment>
            <p style={{
                fontSize: "2rem",
                textAlign: "center",
                margin: "0 0.5rem"
            }}>
                Mathematics is not about numbers,equations,
                computations or algorithms: its about understanding-
                Willian Paul Thurston
            </p>
        </React.Fragment>
    )
}

export default Quote;