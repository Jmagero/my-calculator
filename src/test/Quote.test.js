import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Quote from '../Quote';

it("renders Quote", ()=>{
    const tree = renderer
    .create(<Quote />)
    .toJSON();
    expect(tree).toMatchSnapshot();
})