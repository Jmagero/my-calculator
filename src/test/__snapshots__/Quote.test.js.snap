// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Quote 1`] = `
<p
  style={
    Object {
      "fontSize": "2rem",
      "margin": "0 0.5rem",
      "textAlign": "center",
    }
  }
>
  Mathematics is not about numbers,equations, computations or algorithms: its about understanding- Willian Paul Thurston
</p>
`;
